src/plugins/tables/tables.js

```diff
@@ -158,8 +160,7 @@ $document.on( "draw.dt", selector, function( event, settings ) {
 		// Should be pushed upstream to DataTables
 		$elm.next( ".bottom" ).find( ".paginate_button" )
 			.attr( {
-				"role": "button",
-				"href": "javascript:;"
+				"href": "#" + $elm.context.id
 			} )
 			.not( ".previous, .next" )
 				.attr( "aria-pressed", "false" )
```

Comment:
Please update the markup as per the latest practice. As a reference see the pagination template documentation in the GCWeb search template.